tools/clone_flutter.sh

```diff
@@ -19,9 +19,10 @@ cd $ENGINE_PATH/src/flutter
 # Special handling of release branches.
 ENGINE_BRANCH_NAME=`git branch | grep '*' | cut -d ' ' -f2`
 versionregex="^v[[:digit:]]+\."
+releasecandidateregex="^flutter-[[:digit:]]+\."
```

Comment:
Why not `^flutter-[[:digit:]]+\.[[:digit:]]+-candidate\.[[:digit:]]+$`?

Reply:
makes sense. will do.

Reply:
done.